.github/workflows/categorizer.yml

```
@@ -29,25 +31,55 @@ jobs:
-          echo "label=$CLEAN_LABEL" >> $GITHUB_OUTPUT
+          # Clean and normalize AI output
+          RAW_OUTPUT='${{ steps.ai-categorize.outputs.response }}'
+          CLEANED=$(echo "$RAW_OUTPUT" | \
+            tr '[:upper:]' '[:lower:]' | \
+            sed -e 's/["'\''$$$$]//g' -e 's/[, ]\+/, /g' -e 's/^[ ,]*//' -e 's/[ ,]*$//')
+          
+          echo "raw_labels=$CLEANED" >> $GITHUB_OUTPUT
 
-      - name: Apply label to issue
+      - name: Apply validated labels
         uses: actions/github-script@v6
```

actionlint (1.7.8)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)